packages/vite-config/src/baseConfig.ts

```diff
@@ -1,34 +1,23 @@
-import react from '@vitejs/plugin-react'
-import { resolve, dirname } from 'path'
+import { dirname, resolve } from 'path'
+import { fileURLToPath } from 'url'
 import { defineConfig, UserConfig } from 'vite'
-import svgr from 'vite-plugin-svgr'
 import { nodePolyfills } from 'vite-plugin-node-polyfills'
-import { fileURLToPath } from 'url'
 
 const __filename = fileURLToPath(import.meta.url)
 const __dirname = dirname(__filename)
 
-export const config: UserConfig = {
+/**
+ * Vite config properties that are shared across all Vite configurations.
+ */
+const baseConfig: UserConfig = defineConfig({
   server: { port: 3000 },
   build: {
     outDir: './build',
     commonjsOptions: {
       transformMixedEsModules: true,
     },
   },
-  plugins: [
-    react(),
-    svgr({
-      svgrOptions: {
-        plugins: ['@svgr/plugin-jsx'],
-        ref: true,
-        exportType: 'named',
-      },
-      // Explicitly exclude SVG imports that end in a query (such as ?url) - Vite can already handle these
-      include: /^.*\.svg$/,
```

Comment:
React plugins were erroneously included in the base config, so they were duplicated in the final app configs, causing errors in development mode